processing/src/main/java/io/druid/query/groupby/GroupByQueryEngine.java

```diff
@@ -71,6 +79,62 @@
   private final Supplier<GroupByQueryConfig> config;
   private final StupidPool<ByteBuffer> intermediateResultsBufferPool;
 
+  private static void getBytesFromBuffer(ByteBuffer src, byte[] dst, int srcOffset, int readLen)
```

Comment:
There are bulk array copy methods on ByteBuffer.  Why are we doing this?  If it is for good reason, great, but it deserves a comment 'cause anyone who reads this will likely wonder why we aren't doing the bulk copy.